benchmark-ganache.ts

```diff
@@ -0,0 +1,165 @@
+import * as assert from "assert";
+
+const chalk: any = require("chalk");
+import * as ethers from "ethers";
+import Ganache from "ganache";
+
+const url = "mainnet";
```

Comment:
What happens when we pass in `mainnet` as the URL here, e.g. what RPC URL is being used behind the scenes? I'm thinking we should change this to an env var so all frameworks use the same RPC URL
```suggestion
const url = process.env.ETH_RPC_URL as string;
```

Reply:
I left `"mainnet"` in as backup, so I did this as `|| "mainnet"` instead of your suggestion (also do you even need that `as string` there?)

Reply:
`"mainnet"` uses Ganache's Infura archive node URL. We can't share it openly here, as we don't want to encourage (ab)use of this ID outside Ganache.